### DISCUSSION TITLE
Steam Workshop tags and categories?

### DISCUSSION THREAD

**Discussion by ndelta0** [voxeltycoon/issues#456] [Modding]
Maybe it's a good idea to add tags and categories to the Steam Workshop page?

I'm talking about setting if the mod is script/asset and tags of trains, road vehicles, resources, etc.

**Comment by victorleblais** [comment]
Here's an example (workshop of Left 4 Dead 2):
![image](https://user-images.githubusercontent.com/4365075/115476347-f7dd9500-a241-11eb-880d-d21b8f946d8b.png)


It would be great to make a list of all the categories and items we would like to have.

**Comment by ndelta0** [comment]
Example of Space Engineers' Workshop (1 category and multiple tags)
![image](https://user-images.githubusercontent.com/29150574/115512794-a43f6b80-a282-11eb-8d96-05364f24b324.png)
